src/client/index.js

```diff
@@ -45,10 +37,12 @@ export default function statinamic({
     <div id="statinamic-container">
       <MetadataProvider metadata={ metadata }>
         <Provider store={ store }>
-          <Router history={ history } routes={ routes } />
+          <div>
```

Comment:
Not a fan about the extra div.
FYI, the server side is here: https://github.com/MoOx/statinamic/blob/cb000b78d0172e05dadffeb8bdb0fecbe770/src/static/to-html/url-as-html.js